### ISSUE TITLE
[Suggestion] Add default themes

### ISSUE THREAD

**Issue by gitblep** [nullworks/cathook#921]
Add default preconfigured themes for the user to select
e.g. themes from other cheats, from popular GTK/Qt themes, light/dark versions of default theme

**Comment by BenCat07** [comment]
Themes in the menu or what?

**Comment by BenCat07** [comment]
Adding even one more theme would like require a full recode and redesifn of how the menu works

**Comment by gitblep** [comment]
> Themes in the menu or what?

Yeah. And I really only meant new color schemes, in case it wasn't clear & if it makes a difference.
